src/components/themed/TransactionListTop.tsx

```diff
@@ -159,23 +156,6 @@ export class TransactionListTopComponent extends React.PureComponent<Props, Stat
     this.setState({ lockedNativeAmount })
   }
 
-  handleOpenWalletListModal = () => {
```

Comment:
When was it decided to remove this functionality? Unless there was consent (I had not heard of such), please put it back

Reply:
It was decided earlier on with Madison but has since reverted the decision since we're retaining this functionality in the new design. Will revert